fuel-vm/src/storage.rs

```diff
@@ -31,16 +31,31 @@ impl Mappable for ContractsRawCode {
     type Value = [u8];
 }
 
-/// The storage table for contract's additional information as salt, root hash, etc.
+/// The storage table for contract's additional information
 pub struct ContractsInfo;
```

Comment:
I am not even sure that we need this table here=) Since we don't use salt in any logic, we can simply skip it, and it can be fully off-chain information.